Druid/DRCogitCodeGenerator.class.st

```diff
@@ -197,7 +209,7 @@ DRCogitCodeGenerator >> cogitBranchFromCondition: condition types: branchTypes [
 			          ifFalse: [ 'LessOrEqual:' ].
 		^ branchPrefix , suffix ].
 
-	1 halt
+	self notYetImplemented.
```

Comment:
I hate this method....

Reply:
Oh I just thought it was weird when I ran a test and it stopped in a halt that wasn't mine, but I can leave it if you prefer it (and maybe add a comment?)

Reply:
No, this is ok! I was talking about the `cogitBranchFromCondition: condition types: branchTypes` method 😆

Reply:
ohhh ok jajajaj
yeah, it could definitetly be better, that's waaay too many `ifTrue`